rpc/types_contract.go

```diff
@@ -27,6 +27,25 @@ type EntryPointsByType struct {
 	L1Handler   []EntryPoint `json:"L1_HANDLER"`
 }
 
+type DeprecatedContractClass struct {
```

Comment:
This will probably cause some conflicts with #293 
Also it will need to be fixed to actually use the new `ContractClass` type instead of `ContractClassTemp`
I think we should wait for #293 to get in, then fix this one and merge it, wyt?